irohad/main/irohad.cpp

```diff
@@ -26,50 +26,80 @@
 
 #include "logger/logger.hpp"
 
+/**
+ * Validator for the configuration file path input argument.
+ * Path is considered to be valid if it is not empty.
+ */
 bool validate_config(const char *flag_name, std::string const &path) {
```

Comment:
Doxygen-based info is missing :( like `@param` or `@return`

Reply:
You are right. Here is a new pull request with a fix https://github.com/hyperledger/iroha/pull/882